config/locales/en.yml

```diff
@@ -192,14 +192,14 @@ en:
     errors:
       already_exists: already exists
       must_exist: must exist
-      missing_entity_id: id is required for MSA component
+      missing_entity_id: entityID is required for the MSA component
```

Comment:
the entity is already prepended, so if we accept this change the error will become `Entity entityID is required...` . I'll change it to `ID` so the resulting message becomes `Entity ID is required...`